src/osx/webview_webkit.mm

```diff
@@ -933,6 +964,33 @@ wxFileDialog dlg(webKitWindow->GetParent(), wxFileSelectorPromptStr, "", "",
 }
 #endif // __MAC_OS_X_VERSION_MAX_ALLOWED
 
+// The following WKUIDelegateMethods are undocumented as of macOS SDK 11.0,
+// but are documented in the WebKit cocoa interface headers:
+// https://github.com/WebKit/WebKit/blob/main/Source/WebKit/UIProcess/API/Cocoa/WKUIDelegatePrivate.h
+
+- (void)_webView:(WKWebView *)webView printFrame:(WKFrameInfo*)frame
+{
+    webKitWindow->Print();
+}
+
+- (void)_webViewDidEnterFullscreen:(WKWebView *)webView
+{
+    wxWebViewEvent event(wxEVT_WEBVIEW_FULLSCREEN_CHANGED, webKitWindow->GetId(),
+        webKitWindow->GetCurrentURL(), wxString());
+    event.SetEventObject(webKitWindow);
+    event.SetInt(1);
+    webKitWindow->HandleWindowEvent(event);
+}
+
+- (void)_webViewDidExitFullscreen:(WKWebView *)webView
+{
+    wxWebViewEvent event(wxEVT_WEBVIEW_FULLSCREEN_CHANGED, webKitWindow->GetId(),
+        webKitWindow->GetCurrentURL(), wxString());
+    event.SetEventObject(webKitWindow);
+    event.SetInt(0);
+    webKitWindow->HandleWindowEvent(event);
```

Comment:
I'd extract this into some static `SendFullscreenChangedEvent()` helper function to avoid repeating these few lines, but it's pretty minor, of course.

Reply:
Implemented as a objC method.